core/new-gui/src/app/dashboard/component/top-bar/user-icon/user-login/ngbdmodal-user-login.component.ts

```diff
@@ -30,6 +33,24 @@ export class NgbdModalUserLoginComponent implements OnInit {
     this.detectUserChange();
   }
 
+  signupForms = new FormGroup(
```

Comment:
move this variable definition to above constructor. also why do we not have user name in this formGroup?

Reply:
Done. Made a formGroup that contains all forms and added username to that group.